### PR TITLE
Porting EfficientNets to KerasCV

### PR DESCRIPTION
**Pull request by AdityaKane2001**
@sayakpaul @LukeWood 

Added Efficientnets as discussed in #625.

/auto Closes https://github.com/keras-team/keras-cv/issues/551

### CODE REVIEW

File: keras_cv/models/efficientnet_v1.py [hunk 1]
```diff
@@ -0,0 +1,733 @@
...
+        # Note that the normaliztion layer uses square value of STDDEV as the
+        # variance for the layer: result = (input - mean) / sqrt(var)
+        # However, the original implemenetation uses (input - mean) / var to
+        # normalize the input, we need to divide another sqrt(var) to match the
+        # original implementation.
+        # See https://github.com/tensorflow/tensorflow/issues/49930 for more
+        # details
+        x = layers.Rescaling(1.0 / 255.0)(x)
+        x = layers.Normalization(axis=BN_AXIS)(x)
+        x = layers.Rescaling(1.0 / tf.math.sqrt(IMAGENET_STDDEV_RGB))(x)
```
Comment (LukeWood) [inline keras_cv/models/efficientnet_v1.py:431]: we use a standard rescaling in KerasCV:

Rescaling(1/255.0) - you can drop the normalization and other rescaling.

**Comment by AdityaKane2001** [comment]
@LukeWood could you please take a look again?

**Review by LukeWood — APPROVED** [review]

**Comment by LukeWood** [comment]
Hmm, still timing out.  Gah

**Comment by LukeWood** [comment]
All tests are passing, but CI is broken.  I ran all tests locally to confirm.

**Comment by LukeWood** [comment]
Thanks @AdityaKane2001

**Pull request merged**